packages/playground/website/src/lib/hooks.ts

```diff
@@ -36,10 +40,22 @@ export function usePlayground({ blueprint, storage }: UsePlaygroundOptions) {
 			remoteUrl.searchParams.set('storage', storage);
 		}
 
+		const onBlueprintStepError: OnStepError = (
+			error: any,
+			step: StepDefinition
+		) => {
+			if (step.step === 'installPlugin') {
+				// Gather errors and show in notification
```

Comment:
It's a question what to do here with the gathered errors, before we implement the toast notification.